CHANGELOG.md

```diff
@@ -5,8 +5,11 @@
         - Plugins supported: `table`
     - API:
       - Support for exits numbers and names. New member `exits` in `RouteStep`, based on `junction:ref` on ways
+      - `RouteStep` now has new parameter `classes` that can be set in the profile on each way.
```

Comment:
(unrelated to that maybe add to the changeset that this is about _road classes_ and make an example)